reccmp/tools/roadmap.py

```diff
@@ -33,9 +40,11 @@ class ModuleMap:
     """Load a subset of sections from the pdb to allow you to look up the
     module number based on the recomp address."""
 
-    def __init__(self, pdb, binfile) -> None:
-        cvdump = Cvdump(pdb).section_contributions().modules().run()
-        self.module_lookup = {m.id: (m.lib, m.obj) for m in cvdump.modules}
+    def __init__(self, pdb: Path, binfile: Bin) -> None:
```

Comment:
Could replace `Bin` with `IsleBin` here and condense lines 18 and 19. (This may change after #26 anyway)